src/components/color-picker/color-picker-palette.tsx

```diff
@@ -0,0 +1,104 @@
+import { Component, h, State, Event, EventEmitter, Prop } from '@stencil/core';
+/**
+ * @private
+ */
+
+@Component({
+    tag: 'limel-color-picker-palette',
+    shadow: true,
+    styleUrl: 'color-picker-palette.scss',
+})
+export class PaletteExample {
+    /**
+     *
+     */
+    @Prop()
+    public searchedColor;
+
+    @State()
+    private value: string;
+
+    /**
+     * Emits chosen value to the parent component
+     */
+    @Event()
+    private change: EventEmitter<string>;
+
+    private colors = [
+        'red',
+        'pink',
+        'magenta',
+        'purple',
+        'violet',
+        'indigo',
+        'blue',
+        'sky',
+        'cyan',
+        'teal',
+        'green',
+        'lime',
+        'grass',
+        'yellow',
+        'amber',
+        'orange',
+        'coral',
+        'brown',
+        'gray',
+        'glaucous',
+    ];
+
+    public render() {
+        this.value = this.searchedColor;
```

Comment:
Good to avoid updating the state in the render method. We can create `componentWillLoad` method and assign the state there.

```
  public componentWillLoad() {
        this.value = this.searchedColor;
    }